tools/windows_buildenv.bat

```diff
@@ -150,7 +150,7 @@ EXIT /B 0
 
 :READ_ENVNAME
     ECHO Reading name of prebuild environment from "%MIXXX_ROOT%\packaging\windows\build_environment"
-    SET /P BUILDENV_NAME=<%MIXXX_ROOT%\packaging\windows\build_environment
+    SET /P BUILDENV_NAME=<"%MIXXX_ROOT%\packaging\windows\build_environment"
     SET BUILDENV_NAME=!BUILDENV_NAME:PLATFORM=%PLATFORM%!
     SET BUILDENV_NAME=!BUILDENV_NAME:CONFIGURATION=%CONFIGURATION%!
     SET RETVAL=%BUILDENV_NAME%
```

Comment:
Do we need quotes here as well?

Reply:
Yes, otherwise the batch file demands a user input.